modules/statics/src/base.ts

```diff
@@ -1758,6 +1758,8 @@ export enum UnderlyingAsset {
   'sol-perp' = 'sol-perp',
   'sol-woo' = 'sol-woo',
   'sol-weth' = 'sol-weth',
+  'btc-sollet' = 'btc-sollet',
+  'eth-sollet' = 'eth-sollet',
```

Comment:
Only these tokens to be onboarded for fixing FTX TDs? @Vijay-Jagannathan

Reply:
We already added a few recently. This ticket deals only with a specific set of txns, which need these 2 tokens specifically